### PR TITLE
Update layout for file downloads

### PR DESCRIPTION
**Pull request by teneightfive**
## User Story [#100011112](https://www.pivotaltracker.com/story/show/100011112)

As a bank account admin,
I need the bank admin front-end to allow me to download refund AccessPay files,
So that I can send refunds to the appropriate people.

### CODE REVIEW

**Comment by teneightfive** [comment]
This is an initial pass at this view. We're waiting on results from user testing to see whether we need to implement some form of feedback to the user that the file is being generated to avoid duplicate clicks.

**Comment by SteveMarshall** [comment]
retest this please

**Comment by SteveMarshall** [comment]
This is failing to build because Jenkins can't find v0.3.0 of `money-to-prisoners-common`. @teneightfive does that version need releasing?

**Comment by teneightfive** [comment]
retest this please

**Comment by teneightfive** [comment]
Ah. @SteveMarshall is jenkins still broken? Version [0.3.0](https://github.com/ministryofjustice/money-to-prisoners-common/releases) is now released so this shouldn't fail on that account.

**Comment by teneightfive** [comment]
retest this please

**Comment by SteveMarshall** [comment]
retest this please

**Comment by SteveMarshall** [comment]
retest this please

**Comment by maxf** [comment]
retest this please

**Comment by maxf** [comment]
retest this please

**Comment by maxf** [comment]
@SteveMarshall checks pass now
http://ci.dsd.io/view/Money%20to%20Prisoners/job/money-to-prisoners-bank-admin-build/19
Wanna give it a final look?

File: mtp_bank_admin/templates/bank_admin/dashboard.html [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
-	{% trans "Download ADI Payment Reconciliation File" %}
-</a>
-<br/>
-<a href="{% url "bank_admin:download_adi_refund_file" %}">
-	{% trans "Download ADI Refund Reconciliation File" %}
-</a>
-{% endblock %}
\ No newline at end of file
+
+  <h1>Download files</h1>
```
Comment (SteveMarshall) [inline mtp_bank_admin/templates/bank_admin/dashboard.html]: Wot, no translations?

**Comment by SteveMarshall** [comment]
@maxf Just a few missing translations. Also, might be nice if your commit messages' first lines were in [the imperative mood](http://chris.beams.io/posts/git-commit/#imperative).

**Comment by maxf** [comment]
@SteveMarshall fixed (and more)
Side note: functional tests work in English only. I wonder if it makes sense generally to have i18n-independent tests

**Comment by SteveMarshall** [comment]
@maxf i18n-independent tests would be nice, but later, I think. Good work! Nice to finally get this done! :stars:

**Pull request merged**